docs/docs/tutorials/cost-estimate-report.md

```diff
@@ -0,0 +1,191 @@
+infracost --version
+```
+
+You can next configure the API key for Infracost to enable generating cost reports and dashboards. To do that, run the following command on your terminal:
+
+```sh
+infracost register
+```
+
+Nebari's cost estimate feature requires an API key to be configured before running the tool. If you have configured an API key, run the following command to validate your API key:
+
+```sh
+infracost configure get api_key
+```
+
+We are now ready to run the cost estimate tool.
+
+## Setup initialization
+
+The next step requires us to choose a Cloud Provider to host the project deployment. In this tutorial, we will be using the Google Cloud Platform (GCP). The cloud installation is based on Kubernetes and will be using Terraform to initialize the cloud infrastructure. However, you can follow along to generate the cost report without actually deploying anything.
+
+Follow [these detailed instructions](https://cloud.google.com/iam/docs/creating-managing-service-accounts) to create a Google Service Account with owner-level permissions. After completing the previous step, create and download a JSON credentials file. Store this credentials file in a well-known location and set yourself exclusive permissions.
+
+You can change the file permissions by running the command `chmod 600 <filename>` on your terminal. In this case the environment variables will be such as follows:
+
+```sh
+export GOOGLE_CREDENTIALS="path/to/JSON/file/with/credentials"
+export PROJECT_ID="projectIDName"
+```
+
+The `PROJECT_ID` will be available on the Google Console homepage, under `Project info`.
+
+Create a new directory named `nebari-gcp-deployment` to generate your Nebari deployment configuration:
+
+```sh
+mkdir nebari-gcp-deployment
+cd nebari-gcp-deployment
+```
+
+Install Nebari using `conda`/`pip` by pushing the following command on your terminal:
+
+```sh
+pip install nebari
+```
+
+```sh
+conda install -c conda-forge nebari
+```
+
+To generate a fully-automated configuration file, run the following command on your terminal:
+
+```sh
+nebari init gcp \
+  --project projectname --domain mysite.com \
+  --ci-provider github-actions \
+  --terraform-state=remote \
+  --auth-provider github --disable-prompt
+```
+
+Finally, with the `nebari-config.yaml` created, Nebari can be deployed:
+
+```sh
+nebari deploy -c nebari-config.yaml
+```
+
+It will generate the following directory structure:
+
+```sh
+.
+├── image
+│   ├── Dockerfile.dask-worker
+│   ├── Dockerfile.jupyterhub
+│   ├── Dockerfile.jupyterlab
+│   ├── README.md
+│   ├── dask-worker
+│   ├── docker-bake.json
+│   ├── jupyterhub
+│   ├── jupyterlab
+│   └── scripts
+├── nebari-config.yaml
+└── stages
+    ├── 02-infrastructure
+    ├── 03-kubernetes-initialize
+    ├── 04-kubernetes-ingress
+    ├── 05-kubernetes-keycloak
+    ├── 06-kubernetes-keycloak-configuration
+    ├── 07-kubernetes-services
+    └── 08-nebari-tf-extensions
+```
+
+We can now move ahead to generate the cost estimate.
```

Comment:
I think all of this should be removed. These are installation instructions which are in another section of the docs

Reply:
Just need to make note that this tutorial assumes you have nebari installed.